Dockerfile

```diff
@@ -30,6 +30,7 @@ RUN apt-get update \
     && curl -sL https://deb.nodesource.com/setup_$NODEJS_VERSION | bash \
     && apt-get install -y --no-install-recommends nodejs yarn \
     && npm install -g --unsafe-perm bower gulp phantomjs-prebuilt \
+    && npm install -g eslint \
```

Comment:
I would use the same line (32), like `npm install -g --unsafe-perm bower gulp phantomjs-prebuilt eslint`

Reply:
let me test locally first, because we using `--unsafe-perm` on this line.